.eslintrc

```diff
@@ -4,6 +4,7 @@
     "strict": 0,
     "quotes": [2, "single"]
   },
+  "ignorePatterns": ["dist/", "node_modules/"],
```

Comment:
I found how to fix my issue with the eslint (it was running on the dist and possibly the node_modules folders).